CONTRIBUTING.md

```diff
@@ -32,6 +32,28 @@ _Before_ submitting a pull request, please make sure the following is done…
 
     Note: Replace `<your_username>` with your GitHub username
 
+1. Run Jest with [Docker](https://docs.docker.com/engine/docker-overview/) (Optional) or continue reading for manual instructions.
```

Comment:
ci complains about some linting issues in this file. Could you run `prettier` over it?

Reply:
I ran prettier over this using `yarn run lint:prettier CONTRIBUTING.md` and it fixed spacing on line `35`. Checking if it passes now...